lib/utils.pm

```diff
@@ -1005,9 +1006,9 @@ sub set_zypper_lock_timeout {
     script_run("export ZYPP_LOCK_TIMEOUT='$timeout'");
 }
 
-=head2 workaround_type_encrypted_passphrase
+=head2 unlock_bootloader
 
- workaround_type_encrypted_passphrase();
+ unlock_bootloader();
 
 Record soft-failure for unresolved feature fsc#320901 which we think is
```

Comment:
This is obsolete now and was actually wrong from the beginning

Reply:
Text is still there

Reply:
Thanks, Removed the `record_softfaiure` part then, and all VRs passed!

Reply:
There are some lines of the old description left over

Reply:
Hmm, then remove them and keep one line `Unlock bootloader if boot partition is encrypted` :)